min_stack.py

```diff
@@ -1,3 +1,9 @@
+# Time Complexity : Min Stack: O(1)
+# Space Complexity : O(n)
+# Did this code successfully run on Leetcode : it runs in IDE, but fails in leetcode
```

Comment:
You should not specify size. 
def __init__(self):
        self.stack = []
        self.min_val = sys.maxsize
    def push(self, x: int) -> None:
        if x <= self.min_val:
            self.stack.append(self.min_val)
            self.min_val = x
            self.stack.append(x)